space/stories/space.stories.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { theme } from '~/all';
```

Comment:
I was thinking of calling these `tokens` in my starter kit, both the module and the export
I think in the long run we should make

what is the idea behind naming `theme `? and `all`?

Reply:
`all` is one of the names, that stuck, for the component that combines all the tokens, and @georges-gomes suggested we should use this convention in the long run.
Now we can have a debate about the exported object :)
We can either call it `theme`, `tokens` or `all`. I chose `theme` because this is fed to the `ThemeProvider` component as `theme` prop.
What do you think?

Reply:
Theme is the vocabulary used for system-ui.
We can use something else more appropriated for fast no problem.

Reply:
I understand, but not sure why you decided to rename the module to `all`, that's what caught my eye

FAST seems to be quite special as compared to DS I knew before it, so I don't think it will fit easily into existing standards, and `design tokens` are 1st class citizens there, both in docs and in code, and given it's adaptive, the `theme` is a bit less logical there, bit I'll consider using it there too

Reply:
`design tokens` is the vocabulary in FAST. You should definitely use it.